types/react-native/index.d.ts

```diff
@@ -740,7 +740,8 @@ export interface LayoutChangeEvent {
     }
 }
 
-export interface TextStyleIOS extends ViewStyle {
+export interface TextStyleIOS extends 
+        {
```

Comment:
sorry, this is a mistake, since your repo is too big, i do this change in web browser, and it's hard to edit correctly in such a big file...

I am doing git clone right now, please wait a minute.